content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,142 @@
-description: "WebRTC is a new specification that uses QUIC to offer an alternative to WebSocket. Conceptually, it can be considered WebSocket over QUIC.Learn about WebTransport and how it is used in libp2p."
+description: "WebRTC is a protocol for real-time communication and is used to establish connections between browsers and other nodes."
 weight: 110
 aliases:
     - "/concepts/transports/webrtc"
 ---
 
-Coming soon!
+## What is WebRTC?
+
+[WebRTC (Web Real-Time Communications)](https://webrtc.org/) is a framework for real-time communication
+and is used to establish browser-to-server and browser-to-browser connections between applications.
+WebRTC serves as a good choice for applications that need built-in support for media
+communication and do not have specific requirements for the underlying transport
+protocol.
+
+### Key features for libp2p
+
+
+- Peer connections: WebRTC enables [direct peer-to-peer connections](https://webrtc.org/getting-started/peer-connections)
+  between browsers and other nodes through a process called [signaling](https://webrtc.org/getting-started/peer-connections#signaling).
+  In non-decentralized WebRTC, this is facilitated by a [TURN server.](https://webrtc.org/getting-started/turn-server)
+  This allows for faster and more efficient communication. Peers can also retrieve or consume the media and
+  also produce it.
+
+- Data channel: WebRTC also provides peer-to-peer data channels called
+  [WebRTC data channels](https://developer.mozilla.org/en-US/docs/Games/Techniques/WebRTC_data_channels),
+  which works on
+  [SCTP (Stream Control Transmission Protocol)](https://en.wikipedia.org/wiki/Stream_Control_Transmission_Protocol).
+  A WebRTC data channel allows applications to send text or binary data over an active connection to a peer.
+  Data channels can be [ordered or unordered](https://developer.mozilla.org/en-US/docs/Web/API/RTCDataChannel/ordered),
+  where ordered channels guarantee the message delivery order (like running on TCP), and unordered channels increase
+  speed but deliver messages out of order (like UDP.)
+
+- [NAT traversal](../nat/overview): WebRTC includes mechanisms (like
+  [ICE](https://datatracker.ietf.org/doc/rfc5245/))to connect to nodes that live behind
+  NATs and firewalls.
+
+- [Security](../secure-comm/overview): WebRTC includes built-in security features, and
+  connections are always encrypted by using [DTLS](https://en.wikipedia.org/wiki/Datagram_Transport_Layer_Security)
+  or [SRTP](https://en.wikipedia.org/wiki/Secure_Real-time_Transport_Protocol).
+
+WebRTC includes several APIs to help create a secure connection
+over the web. The
+[`RTCPeerConnection`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/RTCPeerConnection)
+API allows two applications on different endpoints to communicate using a peer-to-peer protocol. WebRTC enables
+a node to send and receive streams that include media content and arbitrary binary data
+through the `DataChannel`.
```

Comment:
This doesn't sound right:
* "a peer-to-peer protocol": this is the protocol we're describing here!
* media data is not sent on streams